src/tigris.ts

```diff
@@ -238,3 +236,52 @@ export class Collection<T extends TigrisCollectionType> {
  * Default instance of the Tigrisclient
  */
 export default new Tigris({serverUrl: `${process.env.TIGRIS_SERVER_URL}`});
+
+export type Filter<T extends string | number | boolean> = {
+    key: string;
+    val: T;
+}
+
+export type LogicalFilter<T extends string | number | boolean> = {
+    logicalOperator: LogicalOperator
+    filters?: Filter<T>[];
+    logicalFilters?: LogicalFilter<T>[];
+}
+
+export const Utility = {
+    filterString<T extends string | number | boolean>(filter: Filter<T>): string {
+        return JSON.stringify(this.filterJSON(filter));
+    },
+
+    filterJSON(filter: Filter<any>): object {
+        const obj = {};
```

Comment:
is there way to fix all the lints warning?

Reply:
Fixed/suppressed some of these warnings and some of these remains there. The linter rules are under tuning.